github/messages_test.go

```diff
@@ -297,7 +297,10 @@ func TestParseWebHook(t *testing.T) {
 			payload:     &DeploymentEvent{},
 			messageType: "deployment",
 		},
-
+		{
```

Comment:
Please move this new block a few lines down, after `DeploymentStatusEvent` to keep them sorted.

Reply:
Done.